reference/conanfile/attributes.rst

```diff
@@ -101,3 +101,24 @@ Controls when the current package built binaries are uploaded or not
 
         class Pkg(ConanFile):
             upload_policy = "skip"
+
+
+required_conan_version
```

Comment:
It's not actually an attribute of the conanfile class? shouldn't this be one level higher?

Reply:
Yeah, I know, but I think as this is the only "global" variable in conanfile.py, it is not worth a dedicated section, it is an "attribute" of the conanfile.py module

Reply:
👍